modules/codegen/test/src/smithy4s/codegen/internals/SmithyBuildSpec.scala

```diff
@@ -30,17 +30,17 @@ final class SmithyBuildSpec extends munit.FunSuite {
     assertEquals(
       actual,
       """|{
-         |    "version": "1.0",
-         |    "imports": [
+         |    "version" : "1.0",
```

Comment:
uPickle and Circe seem to have slightly different opinions on suitable JSON whitespace. I could also change the tests to do a whitespace-ignoring compare to avoid them breaking with changes like this, but how often do we expect to swap out the JSON parsing library? (Famous last words...)